doc/ssl/SSL_CTX_set_keylog_callback.pod

```diff
@@ -0,0 +1,47 @@
+=pod
+
+=head1 NAME
+
+SSL_CTX_set_keylog_callback, SSL_CTX_get_keylog_callback,
+ssl_ctx_keylog_callback - logging TLS key material
+
+=head1 SYNOPSIS
+
+ #include <openssl/ssl.h>
+
+ void(*ssl_ctx_keylog_callback)(const SSL *ssl, const char *line);
```

Comment:
This is meant to be a typedef, methinks.  So:

``` text
 typedef void (*ssl_ctx_keylog_callback)(const SSL *ssl, const char *line);
```

That should fix the doc-nits issue